Cargo.toml

```diff
@@ -123,7 +123,8 @@ ctrlc = "3.2.5"
 dialoguer = "0.10"
 directories = "4.0"
 displaydoc = "0.2"
-ed25519-dalek = "=1.0"
+# ed25519-dalek = "=1.0"
+ed25519-dalek = { version = "2.0", features = ["rand_core", "zeroize"] }
```

Comment:
the version should be fixed "=2.0" like before

Reply:
Done